modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -6780,6 +6817,57 @@ private void _testGetNestedFieldDetailsInRelationships(
 			expectedFieldName, objectFieldNamesAndObjectFieldValues, type);
 	}
 
+	private void _testPatchCustomObjectEntryExternalReferenceCode(
+			String endpoint, String externalReferenceCode)
+		throws Exception {
+
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
+			JSONUtil.put(
+				"externalReferenceCode", externalReferenceCode
+			).toString(),
+			endpoint, Http.Method.PATCH);
+
+		Assert.assertEquals(
+			jsonObject.getString("externalReferenceCode"),
```

Comment:
I think the first parameter should be the expected value to match it with the argument names:

`public static void assertEquals(Object expected, Object actual)`